### PR TITLE
Add u64 flag type

### PR DESCRIPTION
**Pull request by teodor-pripoae**

### CODE REVIEW

**Comment by rnitta** [comment]
thank you for PR, @teodor-pripoae 

the idea seems good, however, this implementation will make inconsistency with `FlagValue::Int(isize)` .
It may well be good to change `FlagValue::Int(isize)` to `FlagValue::Int(i64)` .
any thoughts?

File: src/flag.rs [hunk 1]
```diff
@@ -30,6 +31,7 @@ pub enum FlagValue {
     Bool(bool),
     String(String),
     Int(isize),
+    U64(u64),
```
Comment (ksk001100) [inline src/flag.rs:34]:
```suggestion
    Uint(u64),
```
Rename it because the name is inconsistent with `Int(isize)`

Comment (teodor-pripoae) [reply src/flag.rs]: Done

**Comment by teodor-pripoae** [comment]
Hi,

Any plans to merge this one ? I've made the required changes to rename it to UInt.

**Comment by ksk001100** [comment]
@teodor-pripoae 

Sorry I haven't responded in a long time.
Change u64 to usize to be consistent with Int(isize)

**Comment by teodor-pripoae** [comment]
Ok, changed it.

**Comment by ksk001100** [comment]
@teodor-pripoae Thanks!!

**Pull request merged**